wordpress-mobile-pack.php

```diff
@@ -20,7 +20,7 @@
 function wmobilepack_frontend_init() {
 
     require_once('frontend/class-application.php');
-    new WMobilePack_Application();
+    new WMobilePack_Application(plugin_dir_url(__FILE__));
```

Comment:
You are reusing this function as an argument repeatedly, as per earlier comment, you should cache this to a variable declared in the plugin manifest file.